packages/stores/src/ui-config/manage-liquidity/remove-concentrated-liquidity.ts

```diff
@@ -64,4 +69,31 @@ export class ObservableRemoveConcentratedLiquidityConfig extends TxChainSetter {
 
     return;
   }
+
+  protected get queryPosition(): ObservableQueryLiquidityPositionById {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    return this.queriesStore
+      .get(this.chainId)
+      .osmosis!.queryLiquidityPositionsById.getForPositionId(this.positionId);
```

Comment:
Why the non-null assertion here? Is there any possibility osmosis could be null? The code will break otherwise

Reply:
No it's hardcoded in the root store, which never changes, which we know will be there> I believe the queriesStore.get() fn uses the `Parital<>` type somewhere up the chain. It's a long story, but @Thunnini did some complex typescript wizardry to make the queriesStore extensible with our own queries (contained on that `osmosis` member)